cmake/version.cmake

```diff
@@ -6,9 +6,9 @@ if (EXISTS "${PROJECT_SOURCE_DIR}/version_override.cmake")
     include("${PROJECT_SOURCE_DIR}/version_override.cmake")
 endif()
 
-set_if_not_defined(FSO_VERSION_MAJOR 3)
-set_if_not_defined(FSO_VERSION_MINOR 8)
-set_if_not_defined(FSO_VERSION_BUILD 1)
+set_if_not_defined(FSO_VERSION_MAJOR 20)
```

Comment:
@MageKing17 pointed out, and I agree, that this should be 19.x.y until 20.0 is released.  I suggest setting this to 19.1.0.

Reply:
I remember this having been the original plan but I also proposed that we could continue with 20.1 for nightlies if we wanted to make the first release 20.2.0.  And at some point we would move the date from the revision into the build number for nightlies.  So going forward we could even just change the major version as soon as a new year rolls around regardless of the current nightly version, for instance 20.3 could roll over to 21.1, etc.  Releases would then almost always be y.2, y.4, etc and we'd never use the y.0 for anything, to keep evens as stable and odds as nightlies.

Reply:
Think about it this way: until the 20-RC1, version 20 is not yet valid.  Therefore the nightly builds and other builds should be 19.x.  The first release for 2020 should be 20.0, not 20.2.

Release numbers should signify milestones.  "Posting a release" is a milestone, but simply having the calendar flip from one year to the next is not.  Ideally the major number will match the year, but the major number should not change solely because the year changed.

Reply:
That doesn't make much sense.  The RC will always just be the complete version string then -RC, so it doesn't matter what that version string is as long as we account for that.  The nightlies can still be 20.1.2020mmdd, and the first RC would be 20.2.0-RC1.  The first stable would just be 20.2.0.  20.1.2020mmdd < 20.2.0-RC1 < 20.2.0.  Still works if we don't stick with the first release having to be 20.0.  I suggested the same thing you are saying the other day in Discord the other day but there was pushback it isn't necessary and I admit that there is no real reason the first release has to be 20.0.  It just would have been nice to have this cemented with 19.x and we could have moved to 19 for the first nightlies insead of 3.8.1, but there's still no reason we can't change this again.  I have to admit I feel like being able to freely change the nightlies when we want to the new year, at the start of the year, seems nicer than forcing us to keep the previous version around for them _just_ so the first stable release each year is a yy.0 release.

Reply:
I don't see why we should confuse people by having 20.2 be the first release in 2020.